deliverables/README.md

```diff
@@ -8,61 +11,134 @@ To answer questions, please just put the answer on the next line after the quest
+
+
+**What are elements?**  
+elements make up the web page, tags mark the beginning and end of an element.
+
+**What are tags?**   
+tags help make up HTML, they surround elements and apply meaning to it. 
+
+
+**What are attributes?**   
+attributes are additional information that go in between the tags of an element, these are in quotations and are often values.
+
+
+**What are forms?**  
+Forms define a form that allows user inputted data to be sent somewhere.
+
+
+**What is a div?**  
+divs are used to add extra information to an existing chunk of information in HTML. A div can also be used to group existing sections of code together.
+
+
+**What are selectors?**  
+selectors are the names given to styles in internal and external style sheets. 
+
+**What are properties?**   
+Properties go inside the curly brackets of the selectors. These control specific things in the aesthetics of the web page.
+
+**What are values?**  
+values go after the properties in selectors. These control how much of the selector is happening and are followed by a semi-colon.
+
+**How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?**  
+Styles can be inherited from a parent element to a child element. So if you apply something to the unordered list, all of the list items will have it. 
+
+**What are two CSS attributes you can change to push an element around on the page?**   
+padding and margins 
+
+**What are the three different ways to include CSS in your project or use CSS to style a particular element?**  
+1. link the external styleet to your HTML<br/>
+2. write CSS in your HTML as in on-page style defintions  
+3. inline styles 
+
+
 
 ---
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**   
+A git is a distributed revision control system.
+
+**What is SCM?**  
+Source Code Management system
+
+**What is a VCS?**  
+Version Control System 
+
+**Why is Git useful for a developer?**   
+Git is useful becuase it can handle very large or small projects with efficiancy and speed.  
```

Comment:
Good answer, but was hoping for a bit more depth here.  Git allows us to make sure that we don't lose any work and keep different versions of anything we are working on.